subworkflows/local/annotate_snv/main.nf

```diff
@@ -23,64 +23,65 @@ workflow ANNOTATE_SNV {
     skip_clinical_filter           // parameter: [mandatory] default: true
 
     main:
-        ch_versions = Channel.empty()
+    ch_versions = Channel.empty()
 
-        // Annotate with VEP
-        ENSEMBLVEP_VEP(
-            vcf.map{ meta, vcf -> [ meta, vcf, [] ] },
-            val_vep_genome,
-            "homo_sapiens",
-            val_vep_cache_version,
-            ch_vep_cache,
-            ch_fasta,
-            ch_vep_extra_files
-        )
+    // Annotate with VEP
+    ENSEMBLVEP_VEP(
+        vcf.map{ meta, vcf_ -> [ meta, vcf_, [] ] },
```

Comment:
```suggestion
        vcf.map{ meta, _vcf -> [ meta, _vcf, [] ] },
```
I would use a leading underscore rather than a trailing one for variables in a closure that you want to distinguish from say the input. In python at least that is the convention for variables meant for internal use.